portfolio/src/main/webapp/script.js

```diff
@@ -44,8 +44,10 @@ function showModal() {
   let webButton = document.getElementById("webButton");
   let githubButton = document.getElementById("githubButton");
 
-  // Variables for the skills row
+  // Variables for the more row
   let skillsButton = document.getElementById("skillsButton");
```

Comment:
I think in js for Google it's better to use ' than ". Several instances in this file.

Reply:
Do you know if its appropriate to keep the double quotations for blocks of text? (htmlElement.innerText = "some text")

Reply:
Or should that text also use single quotations?

Reply:
I think it's single quotes throughout.